src/System.Net.Sockets.Legacy/tests/PerformanceTests/SocketPerformanceAPMTests.cs

```diff
@@ -4,21 +4,14 @@
 using Xunit;
 using Xunit.Abstractions;
 
-//
```

Comment:
I think this is actually still true. If running on `DEBUG` or ARM we would need to scale the comparisons otherwise the tests will fail.

We could add this comment instead:
`// TODO: Change the test so that it iterates for a fixed time instead of a fixed iteration count. Comparisons would still be platform dependent. To fix that we could do a preliminary assessment on the machine speed and factor it in for the expected performance.`